CodeMaid/UI/Dialogs/Options/Cleaning/CleaningUpdateDataTemplate.xaml

```diff
@@ -64,6 +66,13 @@
                             <TextBox Style="{DynamicResource TextBoxMultiLine}" Text="{Binding UpdateFileHeaderTypeScript, UpdateSourceTrigger=PropertyChanged}" />
                         </TabItem>
                     </TabControl>
+                    <GroupBox Header="Variables">
```

Comment:
Instead of a GroupBox, there's an existing UI convention of having an italicized text block underneath a TextBox.  Example:
```
<TextBlock FontStyle="Italic" Text="{x:Static p:Resources.PlaceEachRegularExpressionOnASeparateLine}" />
```
In this case I think $SOLUTION$, $PROJECT$ and $FILENAME$ would be independent of internationalization.  Also, why dollar signs instead of percents or other placeholders?